### ISSUE TITLE
Cannot read property 'type' of undefined

### ISSUE THREAD

**Issue by olegcherr** [ProseMirror/prosemirror#1076]
### Issue details

I'm trying to implement a **plain-text** editor, just like a textarea.
I'm experiencing a problem with pasting formatted text to the editor.

1) When the editor is empty, this text isn't being pasted at all.
2) If editor itn't empty, then the error occurs:
```
TypeError
Cannot read property 'type' of undefined
Fitter.mustMoveInline
https://3j9se.csb.app/node_modules/prosemirror-transform/dist/index.es.js:1287:12
Fitter.fit
https://3j9se.csb.app/node_modules/prosemirror-transform/dist/index.es.js:1143:25
replaceStep
https://3j9se.csb.app/node_modules/prosemirror-transform/dist/index.es.js:1086:40
Transaction.Transform.replace
https://3j9se.csb.app/node_modules/prosemirror-transform/dist/index.es.js:1091:14
Transaction.Transform.replaceRange
https://3j9se.csb.app/node_modules/prosemirror-transform/dist/index.es.js:1463:21
TextSelection.replace
https://3j9se.csb.app/node_modules/prosemirror-state/dist/index.es.js:142:8
TextSelection.replace
https://3j9se.csb.app/node_modules/prosemirror-state/dist/index.es.js:239:33
Transaction.replaceSelection
https://3j9se.csb.app/node_modules/prosemirror-state/dist/index.es.js:539:20
doPaste
https://3j9se.csb.app/node_modules/prosemirror-view/dist/index.es.js:3790:103
editHandlers.paste
https://3j9se.csb.app/node_modules/prosemirror-view/dist/index.es.js:3797:5
HTMLDivElement.view.dom.addEventListener.view.eventHandlers.<computed>
https://3j9se.csb.app/node_modules/prosemirror-view/dist/index.es.js:3281:9
```

My schema looks like in [docs](https://prosemirror.net/examples/schema/):

```
new Schema({
  nodes: {
    doc: { content: "text*" },
    text: {},
  }
})
```

BTW: In the docs this error occurs as well:
![image](https://user-images.githubusercontent.com/3730074/87777243-99166580-c842-11ea-8cba-a29e888a44d7.png)

### Steps to reproduce

1) Copy the following 2 paragraphs to clipboard:

FIRST

SECOND

2) Go to: https://3j9se.csb.app or to the source: https://codesandbox.io/s/prosemirror-brackets-bug-3j9se
3) Click on the editor. Try to CTRL+V. **Nothing will be pasted.**
4) Then type `ab`, place the cursor between the letters and paste again. **The error will occur.**

### ProseMirror version

prosemirror-model 1.11.0
prosemirror-state 1.3.3
prosemirror-view 1.15.2

### Affected platforms

- [x] Chrome
- [x] Firefox
- [ ] Internet Explorer
- [x] Other

### Screenshots / Screencast (Optional)

![20200716-185006](https://user-images.githubusercontent.com/3730074/87687349-11c3e600-c79f-11ea-87a6-1fc1a6d32f5c.gif)

**Comment by olegcherr** [comment]
To fix the issue we can add any `block` node into our scheme. Like this:
```
  doc: {
    content: "text*",
  },
  text: {},
  paragraph: {
    content: "text*",
    group: "block",
    parseDOM: [{tag: "p"}],
    toDOM: () => ["p", 0],
  },
```

**Comment by marijnh** [comment]
Attached patch (released as prosemirror-transform 1.2.8) should help with this.
